Doc/library/subprocess.rst

```diff
@@ -839,7 +839,7 @@ Instances of the :class:`Popen` class have the following methods:
    On Windows :meth:`kill` is an alias for :meth:`terminate`.
 
 
-The following attributes are also available:
+The following read-only attributes are also available:
```

Comment:
They are not read-only (you can overwrite them) but doing so after the Popen object was created won't have any impact. Can you make it clearer that this is the situation?

Reply:
Is there a specific term that describes this behavior? I have the below that describes it, but wondering if there's a word that'd replace 'read-only' and still be accurate (thought about 'unalterable' or 'frozen' but those seem too vague).

```suggestion
The following attributes are also available (note that writing to the attributes will not update their values):
```

Reply:
I committed a better wording.

Personally I don't agree with the bug even being filed, not many people misinterpret this. It is quite normal for instances to have public attributes and for code outside of the class not being intended to modify them.

regardless, this should be more clear.